docs/docs/getting-started/attestation-crafting.md

```diff
@@ -76,9 +76,10 @@ $ export CHAINLOOP_ROBOT_ACCOUNT=deadbeef
 
 `chainloop attestation init` supports the following options
 
-- `--token` robot account provided by the SecOps team. Alternatively, you can set the `CHAINLOOP_ROBOT_ACCOUNT` environment variable.
-- `--revision` of the contract (default: `latest`)
-- `--dry-run`; do not store the attestation in the Control plane, and do not fail if the runner context or required env variables can not be resolved. Useful for development (default: `false`)
+- `--token` robot account or api token provided by the SecOps team. Alternatively, you can set the `CHAINLOOP_ROBOT_ACCOUNT` environment variable.
```

Comment:
out of curiosity, do we support `CHAINLOOP_API_TOKEN` env variable for attestations and or CLI in general?

It would be great that if you export that variable you can do the auto-onboarding + attestation. What do you think?